tracing-flame/src/lib.rs

```diff
@@ -235,6 +246,16 @@ where
             out: self.out.clone(),
         }
     }
+
+    pub fn filter_empty(mut self) -> Self {
+        self.config.filter_empty = true;
+        self
+    }
+
+    pub fn collapse_threads(mut self) -> Self {
```

Comment:
Also,, take it or leave it: elsewhere, we tend to prefix this kind of builder function with `with_`, so following that scheme might make it clearer that these are changing a setting? So maybe `with_empty_samples` and `with_threads_collapsed` or something like that?